README.md

```diff
@@ -2,11 +2,11 @@
 
 [![crates.io](https://img.shields.io/crates/v/extism_pdk.svg)](https://crates.io/crates/extism-pdk)
 
-This library can be used to write [Extism Plug-ins](https://extism.org/docs/concepts/plug-in) in Rust.
+This library can be used to write [Extism Plug-ins](https://extism.org/docs/concepts/pdk) in Rust.
```

Comment:
I understand this link is broken but it will land soon. We're still working on it

Reply:
Ah ok, I had a suspicion that may be the case. What do you think about linking here temporarily and having a TODO to update once it lands?  Just for the folks that wind up on this README in the meantime

Reply:
should resolve now, but need to change to `plugin` https://extism.org/docs/concepts/plugin/

Reply:
actually i should fix that on the docs. will cause confusion. one moment

Reply:
updated!